commit 9c999c74389736a5bc0cbdb5a803632d8fb463dd
Author: Daniel Kahn Gillmor <dkg@fifthhorseman.net>
Date:   Tue Sep 2 09:42:15 2014 -0400

    Typo fix for gpg.texi
    
    Originally reported by Jakub Wilk in https://bugs.debian.org/760273

--- a/doc/gpg.texi
+++ b/doc/gpg.texi
@@ -3435,7 +3435,7 @@
 absolute value internally and thus the last year we can represent is
 2105.
 
-@item  Ceation-Date: @var{iso-date}
+@item  Creation-Date: @var{iso-date}
 Set the creation date of the key as stored in the key information and
 which is also part of the fingerprint calculation.  Either a date like
 "1986-04-26" or a full timestamp like "19860426T042640" may be used.
--- a/doc/gnupg1.info
+++ b/doc/gnupg1.info
@@ -2806,7 +2806,7 @@
      intervals, GnuPG uses an absolute value internally and thus the
      last year we can represent is 2105.
 
-Ceation-Date: ISO-DATE
+Creation-Date: ISO-DATE
      Set the creation date of the key as stored in the key information
      and which is also part of the fingerprint calculation.  Either a
      date like "1986-04-26" or a full timestamp like "19860426T042640"
